src/test/java/com/revaturelabs/ask/AskApplicationModelTests.java

```diff
@@ -0,0 +1,614 @@
+package com.revaturelabs.ask;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Test;
+import org.springframework.data.domain.Page;
+import com.revaturelabs.ask.image.Image;
+import com.revaturelabs.ask.question.Question;
+import com.revaturelabs.ask.response.Response;
+import com.revaturelabs.ask.tag.Tag;
+import com.revaturelabs.ask.user.User;
+
+public class AskApplicationModelTests {
+
+  static Tag testTag1;
+
+  static Tag testTag1PostCreate;
+
+  static Tag testTag2;
+
+  static Tag testTag3;
+
+  static List<Tag> tagReturnList;
+
+  static Set<Image> testImageSet1;
+
+  static Set<Image> testImageSet2;
+
+  static Image testImage1;
+
+  static Image testImage2;
+
+  static Response testResponse1;
+
+  static Response testResponse1PostCreate;
+
+  static Response testResponse2;
+
+  static List<Response> responseReturnList;
+
+  static User testUser1;
+
+  static User testUser1PostCreate;
+
+  static User testUser2;
+
+  static Page<User> returnUsersPage;
+
+  static Question testQuestion1;
+
+  static Question testQuestion1PostCreate;
+
+  static Question testQuestion2;
+
+  static Page<Question> returnQuestionsPage;
+
+  static {
```

Comment:
@KRT0335 , what is the purpose of using these static references and the static initializer? Where in this suite do you use the objects they reference? You just appear to create new mock data in each individual test.

Reply:
@KRT0335 , alright. I now see how you've use them, but this abstraction looks a little crazy, no?